resources/js/components/TopicSelect.vue

```diff
@@ -25,7 +24,7 @@
                 required: false
             },
             assigned: {
-                type: Array,
+                type: Object,
```

Comment:
Let's let the `assigned` prop accept both objects and arrays, since the multiselect component will return one or the other. So this will work here: `type: Object|Array`

Reply:
Ahh i wondered why it was set to array but makes sense now, ive pushed that change